genesyscloud/resource_genesyscloud_journey_segment_test.go

```diff
@@ -0,0 +1,104 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v72/platformclientv2"
+)
+
+func TestAccResourceJourneySegmentSession(t *testing.T) {
```

Comment:
These tests only seem to be creating the resource without updating any attributes or using the `Check` field of the test config to check if the correct attributes are set.
They'll need to be updated to be more comprehensive. That's why we create the builder functions for the test resources instead of writing the config into a file.

Reply:
Getting `API Error: 400 - Error validating JourneyPatterns. `attributes.bleki.value` is not a valid attribute for eventType: All Event Types (f3a0203c-a0ca-47e7-9c0d-63006b90b979)` now

Reply:
Sad. :(
I do not understand why does it success on DCA and not in Live. Will try to get a Live account for testing.